_analyzers/token-filters/apostrophe-token-filter.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: default
+title: Apostrophe token filter
+parent: Token filters
+nav_order: 110
+---
+
+# Apostrophe token filter
+
+The `apostrophe` token filter's primary function is to remove possessive apostrophes and anything following them. This can be very useful in analyzing text in languages which rely heavily on apostrophes, such as Turkish, where apostrophes serves to separate the root word from suffixes, including possessive suffixes, case markers, and other grammatical endings.
+
+
+## Example
+
+Following example can be used to create new index `custom_text_index` with custom analyzer configured in `settings` and used in `mappings`.
+
+```
+PUT /custom_text_index
+{
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "custom_analyzer": {
+          "type": "custom",
+          "tokenizer": "standard", # splits text into words
+          "filter": [
+            "lowercase",
+            "apostrophe"
+          ]
+        }
+      }
+    }
+  },
+  "mappings": {
+    "properties": {
+      "content": {
+        "type": "text",
+        "analyzer": "custom_analyzer"
+      }
+    }
+  }
+}
+```
+
+## Check generated tokens
```

Comment:
```suggestion
## Generated tokens
```

Reply:
@kolchfa-aws Thank you for the feedback, thats now updated